airflow/bin/cli.py

```diff
@@ -2182,6 +2225,12 @@ class CLIFactory:
                     'help': "Delete all DB records related to the specified DAG",
                     'args': ('dag_id', 'yes'),
                 },
+                {
+                    'func': show_dag,
+                    'name': 'show',
```

Comment:
@kaxil  Are you okay with that?
Sample command: 
```airflow dags show exampel_gcp_dlp --imgcat``

Reply:
Yes

Reply:
`show` is a string here and we are not using it as a variable like `format` above, so do not have problems with it.

Reply:
Thanks for opinion. I updated this PR. Let's wait for Travis.